### PR TITLE
17 add logos to the plot uob metoffice national centre for atmospheric science

### PR DESCRIPTION
**Pull request by SutarPrasad**
This PR has added logos to all the available plots

### CODE REVIEW

**Comment by gareth-j** [comment]
For me the tops of the logos are cut off.
Screenshot 2023-08-07 at 18 04 23

**Comment by SutarPrasad** [comment]
> For me the tops of the logos are cut off. Screenshot 2023-08-07 at 18 04 23

Is it after downloading or it's getting rendered like that in the webpage?

**Comment by SutarPrasad** [comment]
> > For me the tops of the logos are cut off. Screenshot 2023-08-07 at 18 04 23
> 
> Is it after downloading or it's getting rendered like that in the webpage?

I have shifted the positions of logos little bit below than previous. Please give it a go once again @gareth-j

**Comment by SutarPrasad** [comment]
image

In SAFARI browser. svg logo inside plot is not loading in firefox hence better we go with .png over there

**Review by gareth-j — CHANGES_REQUESTED** [review]
Looks good but can we pull out the images list somewhere so it isn't repeated so many times in the code?

**Comment by SutarPrasad** [comment]
> Looks good but can we pull out the images list somewhere so it isn't repeated so many times in the code?

resolved to use of function instead of list of images.

File: src/util/helpers.js [hunk 1]
```diff
@@ -8,6 +8,27 @@ export function getVisID() {
 export function isEmpty(obj) {
   return Object.keys(obj).length === 0;
 }
+/**
+ * Creates an image object for the layout.
+ *
+ * @param {string} source - The image source URL.
```
Comment (gareth-j) [inline src/util/helpers.js:14]: Looks good, I like the docstring.

**Review by gareth-j — APPROVED** [review]
Looks good.

**Pull request merged**